couch/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": "0.1.0",
   "name": "couch",
   "display_name": "couch",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Do you think this might call for a major version?

Reply:
+1 for a major version bump.